### ISSUE TITLE
VirtualBox on Google Pixelbook

### ISSUE THREAD

**Issue by ghost** [joedefen/crostini-kde-setup#3]
This might be possible? That would broaden the application support to include Windows.

**Comment by joedefen** [comment]
That would be nice, but to my knowledge running VirtualBox VMs is impossible. E.g., see [virtualbox_installation_error_through_linux](https://www.reddit.com/r/Crostini/comments/rrlwim/virtualbox_installation_error_through_linux/).

So, VirtualBox will not work.   But KVM will work somewhat;  that article refers to [Well, I went and installed Windows 10 on my Chromebook](https://chromeunboxed.com/insalling-windows-on-chromebook-crostini-linux-kvm-qemu-virtual-machine/).

On a Pixelbook, you have some chance of a decent experience, but I have no experience to back that up.  Firstly, generally speaking, I have no need for (a slow versions of) Windows.  Secondly, I found running Window on KVM rather painful and wanting  even on native Linux installs.  I cannot image that it solves many problems.

At any rate, the task seems independent of this project although I'd likely wish to be on Debian Unstable to get the latest KVM before I'd start.

Anyhow ... good luck with that (if you dare try) ;-)
